### DISCUSSION TITLE
刷入后界面与截图中的界面不一致

### DISCUSSION THREAD

**Discussion by LonePheasantWarrior** [Jeffery186/newifi3-d2-openwrt#4] [Q&A]
刷好了进去界面相比截图中的难看的1B啊大哥- -
没看见有啥界面设置

**Comment by LonePheasantWarrior** [comment]
啊找到了,卖了个萌XD

**Reply by 2200737807** [reply]
在哪找到的呀

**Reply by Jeffery186** [reply]
> 在哪找到的呀

系统->语言和界面->主题

**Reply by 2200737807** [reply]
![image](https://user-images.githubusercontent.com/71483108/119246586-4c638f80-bbb5-11eb-9d57-2f45fe888fc2.png)
好奇怪刷完就一个主题

**Reply by Jeffery186** [reply]
你下的是Dev分支的吗？Dev分支为了使用Python3环境，节省体积，主要保留了SSR，其他的不太常用的都删了，你想用主题的话可以手动安装bin目录里面有个luci-theme-argon_1.0-2_all.ipk这个软件包；

主分支默认是含argon这个主题的。
